acceptance-tests/src/test/java/tech/pegasys/eth2signer/tests/publickeys/PublicKeysAcceptanceTest.java

```diff
@@ -114,4 +121,23 @@ public void azureKeysReturnAppropriatePublicKey() {
     final Response response = callApiPublicKeysWithoutOpenApiClientSideFilter(SECP256K1);
     validateApiResponse(response, containsInAnyOrder("0x" + PUBLIC_KEY_HEX_STRING));
   }
+
+  @Test
+  public void loadKeys() {
```

Comment:
Can this test be given a more descriptive name to indicate it is about testing loading large number of keys? Perhaps  loadsLargeNumberOfKeys although maybe that's not even good enough 🤔

Reply:
done